be/src/runtime/exec_env.cpp

```diff
@@ -323,6 +323,14 @@ Status ExecEnv::_init(const std::vector<StorePath>& store_paths) {
         _lake_update_manager = new lake::UpdateManager(_lake_location_provider, update_mem_tracker());
         _lake_tablet_manager = new lake::TabletManager(_lake_location_provider, _lake_update_manager,
                                                        config::lake_metadata_cache_limit);
+        if (starrocks::config::starlet_cache_dir.empty()) {
+            for (int i = 0; i < store_paths.size(); ++i) {
```

Comment:
check `src/gutils/strings/join.h`, ease the join of all elements with ':'

something like:
```
config::starlet_cache_dir = JoinStrings(store_paths, ":");
```